### PR TITLE
update to reflect LTS -> Release WG rename

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Review by fhemberger — APPROVED** [review]

File: locale/uk/get-involved/development.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Розробка
+title: Development
```
Comment (lpinca) [inline locale/uk/get-involved/development.md]: Why this has been changed back to English? This is the Ukrainian localization.

**Review by MylesBorins — APPROVED** [review]
LGMT with changes to internationalization fixed

File: locale/uk/get-involved/development.md (hunk 1 above)
Comment (mhdawson) [reply locale/uk/get-involved/development.md]: I had though 'uk' meant united kingdom English.  But sounds like it is actually Ukrainian.  Sorry will remove that change.

**Comment by mhdawson** [comment]
@lpinca pushed commit to fix internationalization

**Review by lpinca — APPROVED** [review]

**Comment by lpinca** [comment]
Thank you.

**Pull request merged**